### ISSUE TITLE
eapd.cms.gov Certs Expire 5/6/22

### ISSUE THREAD

**Issue by cthulhuplus** [CMSgov/eAPD#2211]
Renew the TLS certificates for https://eapd.cms.gov/ and https://eapd-api.cms.gov/

### This task is done when...
 - [x] The TLS certificates for https://eapd.cms.gov/ and https://eapd-api.cms.gov/ have an expires date after 5/6/22 
 - [x] A new ticket/issue/tracking method is created for the next time the the certs need to be renewed

Example ticket:
https://jiraent.cms.gov/browse/CLDSUPSD-6182

**Comment by cthulhuplus** [comment]
Received certs for staging-eapd.cms.gov and staging-eapd-api.cms.gov, still waiting on certs for Prod environment.

**Comment by cthulhuplus** [comment]
Created reminder for May 1 2024 to renew certs for eapd.cms.gov, eapd-api.cms.gov, staging-eapd.cms.gov, and staging-eapd-api.cms.gov
![image.png](https://images.zenhubusercontent.com/5e9749aa5773ead7746338c1/4dc72197-0b09-4118-b507-57dd72e6b7e3)

**Comment by cthulhuplus** [comment]
Created repeating calendar event:
![image.png](https://images.zenhubusercontent.com/5e9749aa5773ead7746338c1/fb58117f-313a-4551-b0f4-4d779131d023)

**Comment by cthulhuplus** [comment]
Created new ZenHub Issue for 2024 certs, tagged it DevOps, and assigned it to myself
![image.png](https://images.zenhubusercontent.com/5e9749aa5773ead7746338c1/548f1c60-8ec6-49e8-81af-8c029f61ceb4)

**Comment by cthulhuplus** [comment]
Working on a GitHub Wiki page for the process since it happens once every 2 years.

**Comment by cthulhuplus** [comment]
https://github.com/CMSgov/eAPD/wiki/Requesting-and-Installing-TLS-Certificates
